frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -208,6 +208,53 @@ if Device:isAndroid() then
     local isAndroid, android = pcall(require, "android")
     if not isAndroid then return end
 
+    -- REMOVE ME BEGINS: This is the interface to request orientation changes on android
+
+    -- all supported modes for API14
+    local orientation_modes = {
+	["portrait"] = "Portrait",
```

Comment:
No need for all that quoting (unless you really want to I suppose :-P)
```suggestion
	portrait = "Portrait",
```